sdk/tdf_config.go

```diff
@@ -118,6 +123,14 @@ func WithKasInformation(kasInfoList ...KASInfo) TDFOption {
 	}
 }
 
+func withSplitPlan(p ...splitStep) TDFOption {
```

Comment:
@dmihalcik-virtru  Is there a reason this is private?

Reply:
For now, I'd rather keep it private until we need to use it externally. This will be used by the public `WithAutoconfigure` option that is being added in https://github.com/opentdf/platform/pull/1051